### DISCUSSION TITLE
How to seed a @db.Time() column?

### DISCUSSION THREAD

**Discussion by henriqueArrazao** [prisma/prisma#12137] [Q&A] [answered]
```
// SCHEMA
class Model {
  id Int @id
  hour DateTime @db.Time()
}

// SEED
await t.model.create({
  data: {
    id: 1,
    hour: "09:00"
  },
});
```

If I put `new Date("2019-01-16 09:00:00")` it works. Is there any way of I don't need to put the whole date?

**Comment by nurul3101** [comment, marked as answer]
Hey @henriqueArrazao 👋 

If you just need to store the hour only as "09:00" you could store it as a string and that will work.

For `@db.Time` attribute you would need to provide a valid ISO formatted date string only.

However as you have provided `@db.Time` attribute only the time will be stored in the database.
For example, providing `new Date("2019-01-16 09:00:00")` in the hour field will only store `09:00:00` in the database.

**Reply by AlwanN01** [reply]
hello @nurul3101 
in my case providing `new Date("2019-01-16 09:00:00")` stored  `02:00:00` in the database (postgres). 
my locale time is `id` or `+07:00`.
is this a bug, or i am missing something?

**Reply by Angeschossen** [reply]
AFAIK prisma saves dates in UTC.

**Reply by vivekjpatel** [reply]
Hello @nurul3101 , 

I see your answer that to provide data type as String instead of DateTime, but at the time of comparing time how to deal with that ? 

Let's say I have stored field startTime as String '11:30:00' and I want to compare with currentTime using where query prisma like 
`where: {
               startTime: {
                                    gt: <Here current time>
                                }
                            }`

Thanks

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
